.github/workflows/scala.yml

```diff
@@ -18,26 +18,30 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        scala: [2.11.12, 2.12.18, 2.13.12]
-        spark: [2.4.8, 3.2.4, 3.3.3, 3.4.1]
+        scala: [2.11.12, 2.12.19, 2.13.13]
```

Comment:
Wouldn't an include syntax be better than exclude in the case of Scala 2.11? 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixinclude

Reply:
I'm going to check and compare, thanks!